core/src/main/java/apoc/trigger/TriggerHandlerNewProcedures.java

```diff
@@ -122,6 +123,14 @@ public static Map<String, Object> dropAll(String databaseName) {
         return previous;
     }
 
+    public static List<Map<String, Object>> getTriggerNodes(String databaseName) {
+        return withSystemDb(tx -> getTriggerNodes(databaseName, tx, null)
+                .stream()
+                .map(TriggerHandlerNewProcedures::toTriggerInfo)
+                .collect(Collectors.toList())
+        );
+    }
```

Comment:
Are you sure this need to be executed in new transaction? 

The only place it's called from is the apoc.trigger.show function which we check is already being executed in a system transaction. Also, none of the other getTriggerNodes functions are executed in a new transaction.

If it's not needed then can we remove it?

Reply:
Yes, we can eliminate the transaction by leveraging the `@Context Transaction tx`.

Anyway, this method only works for this procedure (which has `@Procedure(mode = Mode.READ)`), the others with Mode.WRITE give an error: `"Creating new node label on database 'system' is not allowed for user '' with FULL overridden by READ overridden by READ. See GRANT CREATE NEW NODE LABEL ON DATABASE system "`